k8s/notify-pending-prs.yaml.j2

```diff
@@ -0,0 +1,25 @@
+replicaCount: 1
+image:
+  repository: ghcr.io/chia-network/github-bot
+  tag: {{ DOCKER_TAG }}
+
+deployment:
+  args:
+    - notify-stale
+    - --loop
+
+# Creates a secret with the following values, and mounts as a file into the main deployment container
+secretFile:
+  mountPath: "/config"
+  stringValues:
+    config.yml: |
+      github_token: "{{ BOT_GITHUB_TOKEN }}"
+      internal_team: "{{ INTERNAL_TEAM_NAME }}"
+      check_stale_pending_repos:
+        - name: "Chia-Network/chia-blockchain"
+          minimum_number: 17788
+        - name: "Chia-Network/chia-blockchain-gui"
+          minimum_number: 2300
+      label_skip_users:
```

Comment:
I think these two config keys are actually supposed to be check_repos, and skip_users now

Reply:
fixed